substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/NativeImage.java

```diff
@@ -1078,6 +1079,16 @@ void handleModuleAttributes(Attributes mainAttributes) {
         }
     }
 
+    void handleEnableNativeAccessAttribute(Attributes mainAttributes) {
+        String enableNativeAccess = mainAttributes.getValue("Enable-Native-Access");
+        if (enableNativeAccess != null) {
+            if (!"ALL-UNNAMED".equals(enableNativeAccess)) {
+                NativeImage.showError("illegal value \"" + enableNativeAccess + "\" for Enable-Native-Access manifest attribute. Only ALL-UNNAMED is allowed");
```

Comment:
```suggestion
                throw NativeImage.showError("illegal value \"" + enableNativeAccess + "\" for Enable-Native-Access manifest attribute. Only " + ALL_UNNAMED + " is allowed");
```

Reply:
Out of curiosity, what's the point of `throw` here since `showError` never actually returns. Looking in other `showError` usages the use of `throw` doesn't seem to be consistent.

Reply:
It helps me to see that execution will stop here without knowing the implementation of  `showError`. Of course one can look it up, but I find it more readable. But it is a matter of taste, mostly.